parsers/test/test_US_SPP.py

```diff
@@ -1,9 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
I removed shebangs like these, as they aren't meant to be executable files as pytest test files.